diff --git src/acinclude/ossie.m4 src/acinclude/ossie.m4
index 450c285..19fabfc 100644
--- src/acinclude/ossie.m4
+++ src/acinclude/ossie.m4
@@ -38,11 +38,6 @@ AC_DEFUN([OSSIE_OSSIEHOME],
       fi
     )
 
-    dnl Check if this is a cross, if so prepend the sysroot to the ossie home
-    AS_IF([test "x$cross_compiling" = "xyes"], [
-      CROSS_SYSROOT=`$CC --print-sysroot`
-      ossie_cv_ossie_home=${CROSS_SYSROOT}${ossie_cv_ossie_home}
-    ])
   ])
   AC_SUBST(OSSIE_HOME, $ossie_cv_ossie_home)
 ])
@@ -66,20 +61,12 @@ else
 fi
 ])
 
-dnl use OSSIEHOME as the default prefix unless --prefix is provided
+dnl Changed so that --prefix is ignored as the OSSIEHOME since bitbake passes in --prefix by default
 AC_DEFUN([OSSIE_OSSIEHOME_AS_PREFIX],
 [
-  AS_IF([test "x${prefix}" = "xNONE"], [
-    dnl Prefix wasn't provided, we need to use ossie home
-    AC_REQUIRE([OSSIE_OSSIEHOME])
-    AS_IF([test "x${ossie_cv_ossie_home}" = "xNONE"], [
-      AC_MSG_ERROR([ossie root directory is not set; this is not expected])
-    ])
-    dnl Use ossie home value for prefix
-    ac_default_prefix=${ossie_cv_ossie_home}
-    prefix=${ossie_cv_ossie_home}
-    AC_MSG_NOTICE(using ${ossie_cv_ossie_home} as installation prefix)
-  ])
+  ac_default_prefix=${ossie_cv_ossie_home}
+  prefix=${ossie_cv_ossie_home}
+  AC_MSG_NOTICE(using ${ossie_cv_ossie_home} as installation prefix)
 ])
 
 dnl A variant on OSSIE_SDRROOT for use *only* when OSSIE_OSSIEHOME_AS_PREFIX is being used. Priorities:
