diff --git a/redhawk-codegen/setup.py b/redhawk-codegen/setup.py
index 271d5d6b..4491d1d7 100644
--- a/redhawk-codegen/setup.py
+++ b/redhawk-codegen/setup.py
@@ -43,11 +43,10 @@ if 'build' in sys.argv:
 for arg in sys.argv:
     if '--home' in arg:
         homeSys = True
-        
+
+ 
 if not homeSys and ossiehome != None and not buildArg:
     sys.argv.append('--home='+ossiehome)
-if not ('--old-and-unmanageable' in sys.argv) and not buildArg:
-    sys.argv.append('--old-and-unmanageable')
 
 setup(name='redhawk-codegen',
       version=versions.codegen,
