diff --git src/base/framework/ThreadedComponent.cpp src/base/framework/ThreadedComponent.cpp
--- src/base/framework/ThreadedComponent.cpp
+++ src/base/framework/ThreadedComponent.cpp
@@ -53,7 +53,7 @@ void ProcessThread::run()
         pthread_setname_np(pthread_self(), name.c_str());
     }

-    boost::posix_time::time_duration boost_delay = boost::posix_time::microseconds(_delay.tv_sec*1e6 + _delay.tv_nsec*1e-3);
+    boost::posix_time::time_duration boost_delay = boost::posix_time::microseconds(static_cast<unsigned long>(_delay.tv_sec*1e6 + _delay.tv_nsec*1e-3));
     while (_running) {
         int state;
         try {
