Index: cpp/USRP_UHD.h
===================================================================
--- cpp.orig/USRP_UHD.h
+++ cpp/USRP_UHD.h
@@ -77,8 +77,8 @@ public:
                 return;
             } else if (state == NOOP) {
                 try {
-                    boost::posix_time::time_duration boost_delay = boost::posix_time::microseconds(_delay.tv_sec*1e6 +
-                                                                                                   _delay.tv_nsec*1e-3);
+                    boost::posix_time::time_duration boost_delay = boost::posix_time::microseconds(long(_delay.tv_sec*1e6 +
+                                                                                                   _delay.tv_nsec*1e-3));
                     boost::this_thread::sleep(boost_delay);
                 } catch (boost::thread_interrupted &) {
                     break;
