--- omniORBpy-4.2.0/modules/dir.mk	2013-06-19 09:55:16.000000000 -0400
+++ omniORBpy-4.2.0-patched/modules/dir.mk	2017-01-09 10:17:18.310669203 -0500
@@ -75,9 +75,9 @@
 PYPREFIX  := $(shell $(PYTHON) -c 'import sys; print sys.exec_prefix')
 PYVERSION := $(shell $(PYTHON) -c 'import sys; print sys.version[:3]')
 PYINCDIR  := $(PYPREFIX)/include
-PYINCFILE := "<Python.h>"
-PYINCTHRD := "<pythread.h>"
-DIR_CPPFLAGS += -I$(PYINCDIR)/python$(PYVERSION) -I$(PYINCDIR) -DPYTHON_INCLUDE=$(PYINCFILE) -DPYTHON_THREAD_INC=$(PYINCTHRD)
+PYINCFILE := "<python$(PYVERSION)/Python.h>"
+PYINCTHRD := "<python$(PYVERSION)/pythread.h>"
+DIR_CPPFLAGS += -DPYTHON_INCLUDE=$(PYINCFILE) -DPYTHON_THREAD_INC=$(PYINCTHRD)
 DIR_CPPFLAGS += $(CORBA_CPPFLAGS)
 
 endif
