Index: burstioInterfaces/configure.ac
===================================================================
--- burstioInterfaces.orig/configure.ac
+++ burstioInterfaces/configure.ac
@@ -45,6 +45,10 @@ PKG_CHECK_MODULES([OMNIORB], [omniORB4 >
 PKG_CHECK_MODULES(OSSIE, ossie >= 2.2.5,,exit)
 RH_PKG_IDLDIR([OSSIE], [ossie])
 
+# Adjust the variables from RH_PKG_IDLDIR for the staging area
+OSSIE_IDLDIR="${STAGING_BASE}${OSSIE_IDLDIR}"
+BULKIO_IDLDIR="${STAGING_BASE}${BULKIO_IDLDIR}"
+
 PKG_CHECK_MODULES([BULKIO], [bulkio >= 2.2])
 RH_PKG_IDLDIR([BULKIO], [bulkioInterfaces])
 
@@ -135,7 +139,7 @@ AM_CONDITIONAL(HAVE_JAVASUPPORT, test $H
 # Optionally disable unit tests
 AC_ARG_ENABLE([testing], AS_HELP_STRING([--disable-testing], [disable build of unit tests]))
 AS_IF([test "x$enable_testing" != "xno"], [
-  AM_PATH_CPPUNIT(1.12.1)
+  ifdef([AM_PATH_CPPUNIT], [AM_PATH_CPPUNIT(1.12.1)], [])
   AS_IF([test "x$HAVE_JAVASUPPORT" == "xyes"], [
     dnl Use RPM location hard-coded for now
     AC_SUBST([JUNIT_CLASSPATH], "/usr/share/java/junit4.jar")
