Index: cpp/gpp_setup
===================================================================
--- cpp.orig/gpp_setup
+++ cpp/gpp_setup
@@ -87,10 +87,8 @@ class BaseConfigurator(object):
         self.props['hostName'] = self.hostname
-        tmp_uname_p    = platform.processor()
+        tmp_uname_p    = platform.machine()
         tmp_proc_map   = {'i386':'x86',
-                          'i686':'x86',
-                          'x86_64':'x86_64',
-                          'armv7l':'armv7l'}
-        self.props['processor_name']  = tmp_proc_map.get(tmp_uname_p, 'x86')
+                          'i686':'x86'}
+        self.props['processor_name']  = tmp_proc_map.get(tmp_uname_p, tmp_uname_p)
 
 
 
