--- cpp/configure.ac	2017-07-21 11:00:55.961173854 -0400
+++ cpp/configure.ac.new	2017-07-21 13:21:27.126171563 -0400
@@ -33,7 +33,7 @@
 
 # Dependencies
 PKG_CHECK_MODULES([REDHAWK], [ossie >= 2.0])
-PKG_CHECK_MODULES([PROJECTDEPS],   [fftw3 >= 3.0 fftw3f >= 3.0])
+PKG_CHECK_MODULES([PROJECTDEPS],   [fftw3f >= 3.0])
 RH_SOFTPKG_CXX([/deps/rh/dsp/dsp.spd.xml],[cpp],[2.0])
 OSSIE_ENABLE_LOG4CXX
 AX_BOOST_BASE([1.41])
--- cpp/rh.fftlib.pc.in	2017-07-21 11:00:55.962173862 -0400
+++ cpp/rh.fftlib.pc.in.new	2017-07-21 13:21:17.793117117 -0400
@@ -23,7 +23,7 @@
 
 Name: rh.fftlib
 Description: basic-components signal processing shared library wrapping FFTW
-Requires: rh.dsp >= 2.0 fftw3 >= 3.0 fftw3f >= 3.0
+Requires: rh.dsp >= 2.0 fftw3f >= 3.0
 Version: 2.1.0
-Libs: -L${libdir} -lfftlib -lfftw3 -lfftw3f
+Libs: -L${libdir} -lfftlib -lfftw3f
 Cflags: -I${includedir}
