--- cpp/Makefile.am	2016-03-10 07:09:25.837250321 -0500
+++ cpp-patched/Makefile.am	2016-03-10 07:10:38.069478678 -0500
@@ -25,7 +25,7 @@
 dist_xml_DATA = ../USRP_UHD.scd.xml ../USRP_UHD.prf.xml ../USRP_UHD.spd.xml
 confdir = $(prefix)/dev/devices/rh/USRP_UHD/
 dist_conf_SCRIPTS = ../nodeconfig.py
-ACLOCAL_AMFLAGS = -I m4 -I${OSSIEHOME}/share/aclocal/ossie
+ACLOCAL_AMFLAGS = -I m4
 AUTOMAKE_OPTIONS = subdir-objects
 
 
