Index: src/base/include/ossie/prop_helpers.h
===================================================================
--- src.orig/base/include/ossie/prop_helpers.h
+++ src/base/include/ossie/prop_helpers.h
@@ -84,15 +84,28 @@ namespace ossie
 
     class badConversion : public std::runtime_error {
     public:
-        badConversion(std::string value, std::string type) : std::runtime_error("Unable to perform conversion"), _value(value), _type(type) {};
-        ~badConversion() throw() {};
+        badConversion(std::string value, std::string type) :
+            std::runtime_error(badConversion::error_message(value, type)),
+            _value(value),
+            _type(type)
+        {
+        }
+
+        ~badConversion() throw() {}
+
         virtual const char* what() const throw()
         {
-            std::ostringstream _msg;
-            _msg << std::runtime_error::what() << ": '"<<_value<<"' to type '"<<_type << "'";
-            return _msg.str().c_str();
-        };
+            // This function is preserved to keep the vtable the same, but can
+            // be removed when ABI compatibility is not a concern.
+            return std::runtime_error::what();
+        }
     private:
+        static std::string error_message(const std::string& value, const std::string& type)
+        {
+            std::ostringstream msg;
+            msg << "Unable to perform conversion: '" << value << "' to type '" << type << "'";
+            return msg.str();
+        }
         std::string _value, _type;
     };
 
