Index: burstioInterfaces/src/python/Makefile.am
===================================================================
--- burstioInterfaces.orig/src/python/Makefile.am
+++ burstioInterfaces/src/python/Makefile.am
@@ -69,4 +69,4 @@ clean-local:
 	test ! -h redhawk/burstio || rm redhawk/burstio
 
 install-exec-hook: source-link
-	python setup.py install -f --$(PYTHON_INSTALL_SCHEME)=$(DESTDIR)$(prefix)
+	$(PYTHON) setup.py install -f --$(PYTHON_INSTALL_SCHEME)=$(DESTDIR)$(prefix)
Index: burstioInterfaces/src/python/setup.py.in
===================================================================
--- burstioInterfaces.orig/src/python/setup.py.in
+++ burstioInterfaces/src/python/setup.py.in
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#!/usr/bin/env python2
 #
 # This file is protected by Copyright. Please refer to the COPYRIGHT file 
 # distributed with this source distribution.
