diff --git src/Makefile.am src/Makefile.am
index d5ab063..e8668a0 100644
--- src/Makefile.am
+++ src/Makefile.am
@@ -34,7 +34,7 @@ if HAVE_JAVASUPPORT
   OMNIJNI = omnijni
 endif
 
-SUBDIRS = acinclude etc $(OMNIJNI) base control xml idl testing
+SUBDIRS = acinclude etc $(OMNIJNI) base control xml idl 
 
 # Install makefile fragments
 amdir = $(datadir)/aminclude/redhawk
@@ -43,6 +43,3 @@ am_DATA = aminclude/jarfile.am aminclude/idlj.am
 pkgconfigdir = $(libdir)/pkgconfig
 dist_pkgconfig_DATA = ossie.pc
 
-test:
-	cd testing; ./runtests.py
-
diff --git src/configure.ac src/configure.ac
index 2477ed6..55206a2 100644
--- src/configure.ac
+++ src/configure.ac
@@ -22,7 +22,7 @@ dnl Update this version number immedately after a release
 AC_INIT([ossie],[1.10.1])
 #AM_INIT_AUTOMAKE(nostdinc)
 # allows filenames over 99 characters long during dist
-AM_INIT_AUTOMAKE([1.9 tar-pax])
+AM_INIT_AUTOMAKE([1.9 tar-pax subdir-objects])
 dnl Update these verion numbers immedately before a release
 dnl
 dnl The version number is in the format "current:revision:age"
@@ -274,48 +274,7 @@ AC_CONFIG_FILES(Makefile \
                 base/parser/Makefile \
                 idl/Makefile \
                 xml/Makefile \
-                tools/LogEventAppender/Makefile \
-                testing/Makefile \
-                testing/_unitTestHelpers/buildconfig.py \
-                testing/sdr/dev/devices/ExecutableDevice/Makefile \
-                testing/sdr/dev/devices/BasicTestDevice_cpp/BasicTestDevice_cpp_impl1/Makefile \
-                testing/sdr/dev/devices/BasicTestDevice_java/java/Makefile \
-                testing/sdr/dev/devices/CppTestDevice/cpp/Makefile \
-                testing/sdr/dev/devices/BasicDevWithExecParam_cpp/BasicDevWithExecParam_cpp_impl1/Makefile \
-                testing/sdr/dev/devices/JavaTestDevice/java/Makefile \
-                testing/sdr/dev/devices/SimpleDevice/SimpleDevice_cpp_impl1/Makefile \
-                testing/sdr/dev/devices/props_test_device/props_test_device_cpp_impl1/Makefile \
-                testing/sdr/dev/devices/issue_111_cpp/cpp/Makefile \
-                testing/sdr/dev/devices/issue_111_java/java/Makefile \
-                testing/sdr/dev/devices/PersonaDevice/cpp/Makefile \
-                testing/sdr/dev/devices/ProgrammableDevice/cpp/Makefile \
-                testing/sdr/dev/services/BasicService_java/java/Makefile \
-                testing/sdr/dom/components/CppCallbacks/cpp/Makefile \
-                testing/sdr/dom/components/TestCppProps/Makefile \
-                testing/sdr/dom/components/TestJavaProps/Makefile \
-                testing/sdr/dom/components/TestLegacyJavaProps/Makefile \
-                testing/sdr/dom/components/TestJavaPropsRange/java/Makefile \
-                testing/sdr/dom/components/linkedLibraryTest/Makefile \
-                testing/sdr/dom/components/TestCppsoftpkgDeps/Makefile \
-                testing/sdr/dom/components/javaDep/javaDep/Makefile \
-                testing/sdr/dom/components/HardLimit/HardLimit_java_impl1/Makefile \
-                testing/sdr/dom/components/SimpleComponent/SimpleComponent_cpp_impl1/Makefile \
-                testing/sdr/dom/components/BasicAC/basicac_java_impl1/Makefile \
-                testing/sdr/dom/components/BasicAC/BasicAC_cpp_impl1/Makefile \
-                testing/sdr/dom/components/MessageReceiverCpp/Makefile \
-                testing/sdr/dom/components/MessageSenderCpp/Makefile \
-                testing/sdr/dom/components/EventSend/EventSend_java_impl1/Makefile \
-                testing/sdr/dom/components/EventReceive/EventReceive_java_impl1/Makefile \
-                testing/sdr/dom/components/PropertyChangeEventsCpp/Makefile \
-                testing/sdr/dom/components/PropertyChangeEventsJava/PropertyChangeEventsJava_java_impl1/Makefile \
-                testing/sdr/dom/components/ticket2093/cpp/Makefile \
-                testing/sdr/dom/components/TestComplexProps/cpp/Makefile \
-                testing/sdr/dom/components/TestComplexProps/java/Makefile \
-                testing/sdr/dom/components/TestCppPropsRange/cpp/Makefile \
-                testing/sdr/dom/components/TestAllPropTypes/cpp/Makefile \
-                testing/sdr/dom/components/TestAllPropTypes/java/Makefile \
-                testing/sdr/dom/components/TestLoggingAPI/cpp/Makefile \
-                testing/sdr/dom/components/TestLoggingAPI/java/Makefile )
+                tools/LogEventAppender/Makefile )
 
 AC_OUTPUT
 
