Index: omniORB-4.2.3/mk/python.mk
===================================================================
--- omniORB-4.2.3.orig/mk/python.mk
+++ omniORB-4.2.3/mk/python.mk
@@ -3,14 +3,14 @@
 #############################################################################
 
 PYVERSION := $(shell $(PYTHON) -c 'import sys; sys.stdout.write(sys.version[:3])')
-PYPREFIX  := $(shell $(PYTHON) -c 'import sys; sys.stdout.write(sys.exec_prefix.replace("\\","/"))')
-PYINCDIR  := $(shell $(PYTHON) -c 'import sys, distutils.sysconfig; sys.stdout.write(distutils.sysconfig.get_python_inc().replace("\\","/"))')
+PYPREFIX  := $(PREFIX)
+PYINCDIR  := $(PREFIX)/include
 
 PythonSHAREDLIB_SUFFIX = $(shell $(PYTHON) -c 'import sys, distutils.sysconfig; sys.stdout.write((distutils.sysconfig.get_config_var("SO") or ".so").lstrip("."))')
 
 PY_MODULE_SUFFIX := $(shell $(PYTHON) -c 'import sys; sys.stdout.write((sys.hexversion < 0x3000000) and "module" or "")')
 
-PYINCFILE := "<Python.h>"
+PYINCFILE := "<python$(PYVERSION)/Python.h>"
 PYINCTHRD := "<pythread.h>"
 
 DIR_CPPFLAGS += -I$(PYINCDIR) -DPYTHON_INCLUDE=$(PYINCFILE) -DPYTHON_THREAD_INC=$(PYINCTHRD)
