# Patch provided from http://stackoverflow.com/questions/38571741
# user bhilburn
diff --git src-a/control/sdr/dommgr/ApplicationFactory_impl.cpp src-b/control/sdr/dommgr/ApplicationFactory_impl.cpp
index d370519..92699e0 100644
--- src-a/control/sdr/dommgr/ApplicationFactory_impl.cpp
+++ src-b/control/sdr/dommgr/ApplicationFactory_impl.cpp
@@ -194,7 +194,7 @@ void ApplicationFactory_impl::ValidateSPD(CF::FileManager_ptr fileMgr,
                                           const bool require_prf, 
                                           const bool require_scd) {
   SoftPkg pkg;
-  ValidateSPD(fileMgr, pkg, false, false );
+  ValidateSPD(fileMgr, pkg, sfw_profile, require_prf, require_scd);
 }

 void ApplicationFactory_impl::ValidateSPD(CF::FileManager_ptr fileMgr, 
