--- src/configure.ac	2016-02-19 11:41:51.581688039 -0500
+++ src-patched/configure.ac	2016-02-19 11:45:44.874990007 -0500
@@ -313,66 +313,6 @@
                 idl/Makefile \
                 xml/Makefile \
                 tools/Makefile \
-                tools/LogEventAppender/Makefile \
-                testing/Makefile \
-                testing/_unitTestHelpers/buildconfig.py \
-                testing/sdr/dev/devices/ExecutableDevice/Makefile \
-                testing/sdr/dev/devices/BasicTestDevice_cpp/BasicTestDevice_cpp_impl1/Makefile \
-                testing/sdr/dev/devices/BasicTestDevice_java/java/Makefile \
-                testing/sdr/dev/devices/CppTestDevice/cpp/Makefile \
-                testing/sdr/dev/devices/BasicDevWithExecParam_cpp/BasicDevWithExecParam_cpp_impl1/Makefile \
-                testing/sdr/dev/devices/JavaTestDevice/java/Makefile \
-                testing/sdr/dev/devices/SimpleDevice/SimpleDevice_cpp_impl1/Makefile \
-                testing/sdr/dev/devices/props_test_device/props_test_device_cpp_impl1/Makefile \
-                testing/sdr/dev/devices/issue_111_cpp/cpp/Makefile \
-                testing/sdr/dev/devices/issue_111_java/java/Makefile \
-                testing/sdr/dev/devices/PersonaDevice/cpp/Makefile \
-                testing/sdr/dev/devices/ProgrammableDevice/cpp/Makefile \
-                testing/sdr/dev/devices/cpp_dev/cpp/Makefile \
-                testing/sdr/dev/devices/GPP/cpp/Makefile \
-                testing/sdr/dev/devices/java_dev/java/Makefile \
-                testing/sdr/dev/services/BasicService_java/java/Makefile \
-                testing/sdr/dom/components/CppCallbacks/cpp/Makefile \
-                testing/sdr/dom/components/TestCppProps/Makefile \
-                testing/sdr/dom/components/TestJavaProps/Makefile \
-                testing/sdr/dom/components/TestLegacyJavaProps/Makefile \
-                testing/sdr/dom/components/TestJavaPropsRange/java/Makefile \
-                testing/sdr/dom/components/linkedLibraryTest/Makefile \
-                testing/sdr/dom/components/TestCppsoftpkgDeps/Makefile \
-                testing/sdr/dom/components/javaDep/javaDep/Makefile \
-                testing/sdr/dom/components/java_comp/java/Makefile \
-                testing/sdr/dom/components/HardLimit/HardLimit_java_impl1/Makefile \
-                testing/sdr/dom/components/SimpleComponent/SimpleComponent_cpp_impl1/Makefile \
-                testing/sdr/dom/components/BasicAC/basicac_java_impl1/Makefile \
-                testing/sdr/dom/components/BasicAC/BasicAC_cpp_impl1/Makefile \
-                testing/sdr/dom/components/MessageReceiverCpp/Makefile \
-                testing/sdr/dom/components/MessageSenderCpp/Makefile \
-                testing/sdr/dom/components/EventSend/EventSend_java_impl1/Makefile \
-                testing/sdr/dom/components/EventReceive/EventReceive_java_impl1/Makefile \
-                testing/sdr/dom/components/PropertyChangeEventsCpp/Makefile \
-                testing/sdr/dom/components/PropertyChangeEventsJava/PropertyChangeEventsJava_java_impl1/Makefile \
-                testing/sdr/dom/components/ticket2093/cpp/Makefile \
-                testing/sdr/dom/components/TestComplexProps/cpp/Makefile \
-                testing/sdr/dom/components/TestComplexProps/java/Makefile \
-                testing/sdr/dom/components/TestCppPropsRange/cpp/Makefile \
-                testing/sdr/dom/components/TestAllPropTypes/cpp/Makefile \
-                testing/sdr/dom/components/TestAllPropTypes/java/Makefile \
-                testing/sdr/dom/components/TestLoggingAPI/cpp/Makefile \
-                testing/sdr/dom/components/cpp_comp/cpp/Makefile \
-                testing/sdr/dom/components/TestLoggingAPI/java/Makefile \
-                testing/sdr/dom/components/C1/cpp/Makefile \
-                testing/sdr/dom/components/C2/cpp/Makefile \
-                testing/sdr/dom/components/ECM_CPP/cpp/Makefile \
-                testing/sdr/dom/components/ECM_JAVA/java/Makefile \
-                testing/sdr/dom/components/PropertyChange_C1/cpp/Makefile \
-                testing/sdr/dom/components/PropertyChange_J1/java/Makefile \
-                testing/sdr/dom/components/Property_CPP/cpp/Makefile \
-                testing/sdr/dom/components/commandline_prop/cpp/Makefile \
-                testing/sdr/dom/components/nocommandline_prop/cpp/Makefile \
-                testing/sdr/dom/components/Property_JAVA/java/Makefile \
-                testing/sdr/dom/components/foo/bar/comp/cpp/Makefile \
-                testing/sdr/dom/components/foo/bar/jv/java/Makefile \
-                testing/sdr/dom/components/TestCppOptionalProps/cpp/Makefile \
-                testing/sdr/dom/components/TestJavaOptionalProps/java/Makefile )
+                tools/LogEventAppender/Makefile )
 AC_OUTPUT
 
--- src/Makefile.am	2016-02-19 11:41:50.989704874 -0500
+++ src-patched/Makefile.am	2016-02-19 11:45:44.874990007 -0500
@@ -34,7 +34,7 @@
   OMNIJNI = omnijni
 endif
 
-SUBDIRS = acinclude etc $(OMNIJNI) base control tools xml idl testing
+SUBDIRS = acinclude etc $(OMNIJNI) base control tools xml idl
 
 # Install makefile fragments
 amdir = $(datadir)/aminclude/redhawk
