--- cpp/gpp_setup	2017-01-09 15:15:39.216102148 -0500
+++ cpp-patched/gpp_setup	2017-01-09 15:16:25.909848851 -0500
@@ -90,7 +90,7 @@
                           'i686':'x86',
                           'x86_64':'x86_64',
                           'armv7l':'armv7l'}
-        self.props['processor_name']  = tmp_proc_map.get(tmp_uname_p, 'x86')
+        self.props['processor_name']  = tmp_proc_map.get(tmp_uname_p, 'armv7l')
 
 
 
