diff --git a/src/EventQueue.h b/src/EventQueue.h
index acfb18c..cb6e2ec 100644
--- a/src/EventQueue.h
+++ b/src/EventQueue.h
@@ -72,8 +72,8 @@ public:
   virtual ~EventQueue();
   void setFilter(Filter* filter)
   {
-    auto_ptr<Filter> f(filter);
-    _filter=f; // MS VC++ 6 has no auto_ptr::reset()
+    unique_ptr<Filter> f(filter);
+    _filter=std::move(f); // MS VC++ 6 has no unique_ptr::reset()
   }
 
   inline void append(CORBA::Any* event)
@@ -93,7 +93,7 @@ private:
   long             _next;
   long             _size;
   CORBA::Any**     _queue;
-  auto_ptr<Filter> _filter;
+  unique_ptr<Filter> _filter;
 
   friend class Reader;
 };
