diff -r -u omniEvents-2_6_2/examples/pullcons.cc omniEvents-2_6_2-patch/examples/pullcons.cc
--- omniEvents-2_6_2/examples/pullcons.cc	2005-04-13 11:37:08.000000000 -0400
+++ omniEvents-2_6_2-patch/examples/pullcons.cc	2012-05-02 16:29:29.000000000 -0400
@@ -148,6 +148,8 @@
 #  include <signal.h>
 #endif
 
+#include <cstdio>
+
 #include "CosEventComm.hh"
 #include "CosEventChannelAdmin.hh"
 #include "naming.h"
diff -r -u omniEvents-2_6_2/examples/pullsupp.cc omniEvents-2_6_2-patch/examples/pullsupp.cc
--- omniEvents-2_6_2/examples/pullsupp.cc	2005-04-13 11:37:08.000000000 -0400
+++ omniEvents-2_6_2-patch/examples/pullsupp.cc	2012-05-02 16:30:02.000000000 -0400
@@ -144,6 +144,8 @@
 #  include <signal.h>
 #endif
 
+#include <cstdio>
+
 #include "CosEventComm.hh"
 #include "CosEventChannelAdmin.hh"
 #include "naming.h"
diff -r -u omniEvents-2_6_2/examples/pushcons.cc omniEvents-2_6_2-patch/examples/pushcons.cc
--- omniEvents-2_6_2/examples/pushcons.cc	2005-06-28 14:52:37.000000000 -0400
+++ omniEvents-2_6_2-patch/examples/pushcons.cc	2012-05-02 16:37:35.000000000 -0400
@@ -159,6 +159,8 @@
 #  include <signal.h>
 #endif
 
+#include <cstdio>
+
 #include "CosEventComm.hh"
 #include "CosEventChannelAdmin.hh"
 #include "naming.h"
diff -r -u omniEvents-2_6_2/examples/pushsupp.cc omniEvents-2_6_2-patch/examples/pushsupp.cc
--- omniEvents-2_6_2/examples/pushsupp.cc	2005-04-13 11:37:08.000000000 -0400
+++ omniEvents-2_6_2-patch/examples/pushsupp.cc	2012-05-02 16:45:04.000000000 -0400
@@ -152,6 +152,8 @@
 #  include <signal.h>
 #endif
 
+#include <cstdio>
+
 #include "CosEventComm.hh"
 #include "CosEventChannelAdmin.hh"
 #include "naming.h"
diff -r -u omniEvents-2_6_2/src/main.cc omniEvents-2_6_2-patch/src/main.cc
--- omniEvents-2_6_2/src/main.cc	2005-05-11 17:56:09.000000000 -0400
+++ omniEvents-2_6_2-patch/src/main.cc	2012-05-02 16:23:41.000000000 -0400
@@ -39,6 +39,8 @@
 #  include "getopt.h"
 #endif
 
+#include <cstdlib>
+
 #include "main.h"
 #include "omniEvents.h"
 #include "naming.h"
diff -r -u omniEvents-2_6_2/tools/eventc.cc omniEvents-2_6_2-patch/tools/eventc.cc
--- omniEvents-2_6_2/tools/eventc.cc	2005-05-11 14:44:31.000000000 -0400
+++ omniEvents-2_6_2-patch/tools/eventc.cc	2012-05-02 16:30:36.000000000 -0400
@@ -184,6 +184,8 @@
 #  include <signal.h>
 #endif
 
+#include <cstdio>
+
 #include "omniEvents.hh"
 #include "naming.h"
 
diff -r -u omniEvents-2_6_2/tools/eventf.cc omniEvents-2_6_2-patch/tools/eventf.cc
--- omniEvents-2_6_2/tools/eventf.cc	2005-05-11 14:44:31.000000000 -0400
+++ omniEvents-2_6_2-patch/tools/eventf.cc	2012-05-02 16:31:01.000000000 -0400
@@ -46,6 +46,8 @@
 #  include <iostream.h>
 #endif
 
+#include <cstdio>
+
 #ifdef HAVE_STD_IOSTREAM
 using namespace std;
 #endif
diff -r -u omniEvents-2_6_2/tools/rmeventc.cc omniEvents-2_6_2-patch/tools/rmeventc.cc
--- omniEvents-2_6_2/tools/rmeventc.cc	2005-05-11 14:44:31.000000000 -0400
+++ omniEvents-2_6_2-patch/tools/rmeventc.cc	2012-05-02 16:31:24.000000000 -0400
@@ -46,6 +46,8 @@
 #  include <iostream.h>
 #endif
 
+#include <cstdio>
+
 #ifdef HAVE_STD_IOSTREAM
 using namespace std;
 #endif
